README.md

```diff
@@ -185,7 +185,9 @@ Want to see a more advanced example? [Checkout our test micro kernel](Tests/Fixt
 dunglas_action:
     autodiscover:         # Autodiscover action classes stored in the configured directory of bundles and register them as service.
         enabled:   true
-        directory: Action # The directory name to autodiscover in bundles.
+        directories: # The directories name to autodiscover in bundles.
+            action: [ Action ]
+            command: [ Command ]
```

Comment:
The only case I see where it could fail is if a bundle registers a command with constructor arguments that can't be autowired. 
I personally never saw that (for now the commands mostly use the container) so I think we can keep using the old directory. 

But if you think that case is enough to use a different directory, we can search another name ☺